pages/national_resources/se_resources.md

```diff
@@ -56,29 +72,71 @@ national_resources:
       Your_domain: [human_data]
       Your_tasks: [sensitive]
     url: https://data-guidelines.scilifelab.se/topics/research-involving-human-data/
-  - name: Federated EGA Sweden node
-    description: Secure archiving and sharing of genetic and phenotypic data resulting from Swedish biomedical research projects.
+  - name: FEGA Sweden
+    description: The Swedish node of Federated European Genome-phenome Archive (FEGA), which offers secure archiving and sharing of genetic and phenotypic data resulting from Swedish biomedical research projects.
     instance_of: the-european-genome-phenome-archive
     related_pages:
       Your_domain: [human_data]
-      Your_tasks: [sensitive, existing_data, data_publication]
+      Your_tasks: [sensitive, existing_data, gdpr_compliance, data_publication]
     url: https://fega.nbis.se/
+  - name: Swedish Reference Genome Portal
+    description: A web platform for aggregating, sharing, and visualising non-human eukaryotic genome assemblies and genome annotations (co-)produced by researchers affiliated with Swedish institutions.
+    related_pages:
+      Your_domain: [biodiversity]
+      Your_tasks: [existing_data, data_publication]
+    url: https://genomes.scilifelab.se/
+  - name: SciLifeLab Serve
+    description: A platform offering machine learning model serving, app hosting, and other web tools.
+    how_to_access: Available to life science researchers affiliated with a Swedish research institute
+    url: https://serve.scilifelab.se/
+  - name: SciLifeLab Precision Medicine Portal
+    description: The PM Portal is a service for researchers in the precision medicine field, designed to support and accelerate data-driven life science research in Sweden. It provides links to various data sources, customised dashboards, and resources for navigating data management challenges. Researchers can also find guidance on handling sensitive data and links to relevant tools and services.
+    how_to_access: Available to life science researchers affiliated with a Swedish research institute
+    related_pages:
+      Your_domain: [health_data, human_data]
+      Your_tasks: [existing_data, gdpr_compliance]
+    url: https://precision-medicine-portal.scilifelab.se/
+  - name: Swedish Biodiversity Data Infrastructure - SBDI
+    description: The Swedish Biodiversity Data Infrastructure (SBDI) is a national research infrastructure that provides access to biodiversity data and related services for researchers in Sweden. SBDI aims to facilitate the use of biodiversity data for research, conservation, and sustainable development.
```

Comment:
and in description, you can replace full name with SBDI or keep full name as is but remove "(SBDI)" to avoid repetitiveness.